includes/fields/class-evf-field-range-slider.php

```diff
@@ -0,0 +1,29 @@
+<?php
+/**
+ * Range Slider field
+ *
+ * @package EverestForms\Fields
+ * @since   1.7.0
```

Comment:
Need 1.6.7 as we will be shipping this in the next update.

Reply:
Resolved with 634343b0a6a3660e047a9576e5a86d6b843761fa.